packages/renderer/src/lib/table/table.ts

```diff
@@ -34,6 +34,14 @@ export interface ColumnInformation<Type> {
    */
   readonly width?: string;
 
+  /**
+   * Map the source object to another type for rendering. Allows
+   * easier reuse and sharing of renderers by converting to simple
+   * types (e.g. rendering 'string' instead of 'type.name') or
+   * converting to a different type.
+   */
+  readonly renderMapping?: (object: Type) => any;
```

Comment:
It needs to be generic since the column could be converting to whatever type the renderer supports, but tbh I am still learning typescript and struggling with how you could declare this, help or pointers appreciated.

Reply:
Figured it out, and now I'm very pleased with Typescript. :) Columns now have a second type, which defaults to the first if you're not using this feature. If you are, the renderMapping must convert to this second type.